doc/developer/standalone.txt

```diff
@@ -0,0 +1,109 @@
+Reducing OpenChange dependency on Samba
+=======================================
+
+Introduction
+------------
+
+OpenChange originally started out as a friendly fork as Samba. Later - to reduce
+maintenance overhead tracking Samba as well as packaging - it moved to
+just link against Samba and be a plugin for Samba.
+
+The main reason for relying so heavily on Samba originally was that it
+was an existing MS RPC implementation and communicating with an Exchange server
+at the time was only possible by speaking MS RPC.
+
+Dependencies on Samba
+---------------------
+
+OpenChange currently depends on Samba for:
+
+ * DCE/RPC client/server interaction
+ * ROP marshalling
+ * Various utility functions
+ * Logging
+ * Configuration handling (from smb.conf)
+
+Running OpenChange without DCE/RPC
+----------------------------------
+
+With changes in recent years, in particular with the introduction of EWS and MAPI over
+HTTP, it is now mostly technically possible to build an Exchange server without DCE/RPC,
```

Comment:
I would rename MAPI over HTTP to MAPIHTTP directly to remove ambiguity with MAPI over RPC over HTTP and be closer to the name used in Exchange specifications.